docs/docs/rest-api/public/api/v2/types/pod.raml

```diff
@@ -12,6 +12,7 @@ uses:
   volumes: volumes.raml
   unreachableStrategy: unreachableStrategy.raml
   killSelection: killSelection.raml
+  residency: residency.raml
```

Comment:
Please strongly consider omitting this from the pods API.

Reply:
Done. Will be updated by the very next push.

Reply:
🙇